### PR TITLE
Add information about octavia upgrades impact on Kuryr SDN

### PR DESCRIPTION
**Pull request by luis5tb**
This PR adds information about the octavia upgrades problems with using Kuryr SDN, depending on the way OSP handles them

### CODE REVIEW

**Comment by maxwelldb** [comment]
@luis5tb And this one is 4.5+, right?

**Comment by luis5tb** [comment]
> @luis5tb And this one is 4.5+, right?

right!

File: modules/installation-osp-kuryr-known-limitations.adoc [hunk 1]
```diff
@@ -59,3 +59,41 @@ In Go versions 1.13 and later, TCP is used automatically if DNS resolution using
...
+action is triggered, the existing Security Group rules will be deleted during
+the upgrade and existing Network Policies associated to the related
+Services/Load Balancers will not be enforced. This requires either a Service or
+a Network Policy recreation by the user. It is therefore recommended that the
+{rh-openstak} Operator does not upgrade the existing Load Balancers for the
+{product-title} when Network Policies are used.
+
+* If the {rh-openstack} operator does not handle the upgrade of existing Load
+Balancer VMs, new features enabled by Octavia API (e.g., UDP listeners) won't
+work on the existing load balancers. Given that UDP services where not supported
```
Comment (MaysaMacedo) [inline modules/installation-osp-kuryr-known-limitations.adoc:94]: typo: were

File: modules/installation-osp-kuryr-known-limitations.adoc [hunk 2]
```diff
@@ -59,3 +59,41 @@ In Go versions 1.13 and later, TCP is used automatically if DNS resolution using
...
+in {rh-openstack} versions earlier than 16, if Kuryr SDN is used and the
+operators is not handling the load balancer upgrades, users need to recreate
+their Services if they want to expose UDP ports, instead of modifying the
+existing ones to add UDP ports.
```
Comment (MaysaMacedo) [inline modules/installation-osp-kuryr-known-limitations.adoc:98]: perhaps better to highlight that it's a modification on the service specification

File: modules/installation-osp-kuryr-known-limitations.adoc [hunk 3]
```diff
@@ -59,3 +59,41 @@ In Go versions 1.13 and later, TCP is used automatically if DNS resolution using
...
+
+Both actions have implications when using Kuryr SDN:
+* If the {rh-openstack} operator handles the upgrade, there may be problems
+related to Network Policies enforcement when upgrading from {rh-openstack}
+versions before 16 to {rh-openstack} 16. The main reason is that Octavia
+provides in {rh-openstack} version 16 a new API to handle the access
+restriction to the Load Balancer in a standard way. This is incompatible with
+the previous mechanism that Kuryr uses to restrict the access in {rh-openstack}
+versions earlier than 16. Since Kuryr SDN has no visibility on when the failover
+action is triggered, the existing Security Group rules will be deleted during
```
Comment (MaysaMacedo) [inline modules/installation-osp-kuryr-known-limitations.adoc:85]: Even though it's really likely that this will happen, have we check it?

**Review by MaysaMacedo — COMMENTED** [review]
Looks great! Just few nits and a question.

File: modules/installation-osp-kuryr-known-limitations.adoc (hunk 3 above)
Comment (luis5tb) [reply modules/installation-osp-kuryr-known-limitations.adoc:85]: No, I'll do as soon as I get access to an env where I can test it

**Comment by MaysaMacedo** [comment]
Looks good. Should we add a note about the DNS service recreation on the CNO?

**Comment by luis5tb** [comment]
> Looks good. Should we add a note about the DNS service recreation on the CNO?

done

**Comment by maxwelldb** [comment]
@luis5tb Let me know what you think about 922542b. No rush.

**Comment by luis5tb** [comment]
> @luis5tb Let me know what you think about [922542b](https://github.com/openshift/openshift-docs/commit/922542b9379b352d98553135182ba1bcd5f19ce8). No rush.

Looks good

**Comment by maxwelldb** [comment]
@luis5tb TY.

@rlobillo Would you be able to review these changes?

**Comment by maxwelldb** [comment]
Tweaked to fix parallel construction error. Nothing substantive.

**Review by maxwelldb — APPROVED** [review]

**Comment by bmcelvee** [comment]
LGTM. Don't forget to squash your commits. :)

**Comment by maxwelldb** [comment]
> LGTM. Don't forget to squash your commits. :)

Always! Thanks!

**Pull request merged**